hack/update-codecgen.sh

```diff
@@ -52,7 +98,9 @@ for generated_file in ${generated_files}; do
 	rm -f "${generated_file}"
```

Comment:
Gah, github isn't letting me open this on the right line, but on line 97 there's a bug:

> `for generated_file in ${generated_files}; do`

You converted `generated_files` to an array, but this syntax is now wrong.